<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>regbank_reader_main</class>
 <widget class="QMainWindow" name="regbank_reader_main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>793</width>
    <height>620</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QFormLayout" name="formLayout_2">
        <item row="0" column="0">
         <widget class="QPushButton" name="target_button">
          <property name="minimumSize">
           <size>
            <width>151</width>
            <height>31</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>151</width>
            <height>31</height>
           </size>
          </property>
          <property name="text">
           <string>No target available</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="target_selection">
          <property name="minimumSize">
           <size>
            <width>251</width>
            <height>31</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>251</width>
            <height>31</height>
           </size>
          </property>
          <property name="editable">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="regbank_button">
          <property name="minimumSize">
           <size>
            <width>151</width>
            <height>31</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>151</width>
            <height>31</height>
           </size>
          </property>
          <property name="text">
           <string>No regbank loaded</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="regbank_selection">
          <property name="minimumSize">
           <size>
            <width>251</width>
            <height>31</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>251</width>
            <height>31</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Regbank Sheet ?</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="register_sheet_selection">
          <property name="minimumSize">
           <size>
            <width>231</width>
            <height>31</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>231</width>
            <height>31</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Register ?</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="register_selection">
          <property name="minimumSize">
           <size>
            <width>231</width>
            <height>31</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>231</width>
            <height>31</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="line_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QFormLayout" name="formLayout_3">
        <property name="fieldGrowthPolicy">
         <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
        </property>
       </layout>
      </item>
      <item>
       <layout class="QFormLayout" name="formLayout_4">
        <property name="fieldGrowthPolicy">
         <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
        </property>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTabWidget" name="registers_tab_widget">
      <property name="minimumSize">
       <size>
        <width>773</width>
        <height>304</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>-1</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>793</width>
     <height>29</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionLoad_Regbank">
   <property name="text">
    <string>Load Regbank</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
